tests/unit/moduleapi/async_rm_call.tcl

```diff
@@ -157,6 +158,21 @@ start_server {tags {"modules"}} {
         assert_equal [$rd read] {l 1}
         assert_equal [$rd read] {PONG}
     }
+
+    test {blocking RM_Call abort} {
+        r flushall
+        set rd [redis_deferring_client]
+
+        $rd do_rm_call_async blpop l 0
+        wait_for_blocked_client
+
+        r client kill TYPE NORMAL
+        assert_error {*error reading reply*} {$rd read}
```

Comment:
maybe add an assert that we have 0 blocked clients?
or actually wait for the blocked clients to drop to 0 (`wait_for_blocked_clients_count`)